apps/token-manager/app/src/components/AutoComplete/AutoComplete.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useRef, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Transition, animated } from 'react-spring'
+import { ButtonBase, TextInput, springs, theme, unselectable } from '@aragon/ui'
+import { useClickOutside, useOnBlur } from '../../hooks'
+import IconMagnifyingGlass from './IconMagnifyingGlass'
+
+const { accent, contentBackground, contentBorder, textPrimary } = theme
+const identity = x => x
+
+const AutoComplete = React.memo(
+  React.forwardRef(function AutoComplete(
+    {
+      itemButtonStyles,
+      items,
+      onSelect,
+      onChange,
+      renderItem,
+      required,
+      value,
+      wide,
+    },
+    ref
+  ) {
+    const [opened, setOpened] = useState(true)
+    const wrapRef = useRef()
+
+    const handleClose = useCallback(() => setOpened(false))
+    const handleFocus = useCallback(() => setOpened(true))
+    const handleSelect = useCallback(
+      item => e => {
+        e.preventDefault()
+        onSelect(item)
+      },
+      [onSelect]
+    )
+    const handleChange = useCallback(
+      ({ target: { value } }) => onChange(value),
+      [onChange, value]
```

Comment:
Why is `value` a dependency for `useCallback`?

Reply:
It shouldn't, good catch.